src/test/java/org/apache/commons/text/StringEscapeUtilsTest.java

```diff
@@ -319,6 +319,12 @@ public void testUnescapeHtml4() {
         assertEquals("Hello&##;World", StringEscapeUtils.unescapeHtml4("Hello&##;World"));
     }
 
+    @Test
+    public void testEscapeHtml4Apostrophe() {
+        String unescapedChar = StringEscapeUtils.unescapeHtml4 ("&#39;");
+        assertEquals ("&#39;",unescapedChar, StringEscapeUtils.escapeHtml4 (unescapedChar));
```

Comment:
This test must fail.

Reply:
Yeah, it failed in the Travis build.
org.opentest4j.AssertionFailedError: ' ==> expected: <&#39;> but was: <'>

Reply:
The test case is correct. `assertEquals ("&#39;", StringEscapeUtils.escapeHtml4 (unescapedChar));`